docs/shell-tips/index.md

```diff
@@ -33,3 +33,29 @@ sudo -u $(stat . -c %U) -- wp --path="$PWD" "$@"
 Explanation
 
 The `stat` command returns the owner of the current directory, WordPress root.
+
+## Install and Configure WordPress with WP-CLI
+
+```bash
+wp_install () 
+{ 
+    wp core download --path=$1;
+    cd $1;
+    read -p 'name the database:' dbname;
+    wp core config --dbname=$dbname --dbuser=root --dbpass=awoods --dbhost=localhost;
```

Comment:
@andrewwoods I'm not sure it makes sense to include these hardcoded values. Can you clarify your intent?